manifests/0000_20_etcd-operator_04_clusterrolebinding.yaml

```diff
@@ -11,4 +11,4 @@ roleRef:
 subjects:
```

Comment:
can't comment on the right line, but I thought the goal was to get rid of the cluster-admin role